code/src/components/InputText.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const InputText = ({ question, value, setName }) => {
+	return (
+		<>
+			<label
+				htmlFor="InputText"
```

Comment:
No need to use React Fragment (`</>`) if you return only one element (in your case it's `<label />`)

Reply:
True, I have removed them now.